Makefile

```diff
@@ -254,7 +254,10 @@ test-unit: .init build
 	$(DOCKER_CMD) go test -race $(UNIT_TEST_FLAGS) \
 	  $(addprefix $(SC_PKG)/,$(TEST_DIRS)) $(UNIT_TEST_LOG_FLAGS)
 
-test-integration: .init $(scBuildImageTarget) build
+build-integration: .generate_files
+	$(DOCKER_CMD) go test -race github.com/kubernetes-incubator/service-catalog/test/integration/... -c
```

Comment:
if we add `-i` here, will that cause the integration test to reuse the `pkg/` cache when/if it hits the later rebuild in the `integration.sh` ?

Reply:
Good q, i will tal in my next work session

Reply:
Any follow up thoughts or should we merge and figure it out later?